File: 백준/Bronze/2581. 소수/소수.py
def is_prime(n):
    i = 5

    if n % 2 == 0:
        return n == 2

    if n % 3 == 0:
        return n == 3

    while i * i <= n:
        if n % i == 0:
            return False

        i += 2

    return n != 1

sum = 0

for i in range(int(input()), int(input()) + 1):
    if is_prime(i):
        if sum == 0:
            minimum = i

        sum += i

if sum:
    print(sum)
    print(minimum)
else:
    print(-1)

File: 백준/Bronze/2581. 소수/소수.c
#include <stdio.h>

int main()
{
    int M, N, sum = 0, minimum = 0;
    scanf("%d\n%d", &M, &N);
    
    for (int i = M; i <= N; i++) {
        for (int j = 2; j <= i; j++) {
            if (i % j == 0) break;
            
            if (j == i - 1) {
                if (sum == 0) minimum = i;
                
                sum += i;
            }
        }
        
        if (i == 2) {
            sum += i;
            minimum = i;
        }
    }
    
    if (minimum == 0) printf("-1");
    else printf("%d\n%d", sum, minimum);
}